SWADroid/src/main/res/layout/button_list.xml

```diff
@@ -0,0 +1,213 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
+    <include layout="@layout/progress_screen" />
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical"
+        android:id="@+id/buttonsLayout">
+
+        <ToggleButton
+            android:id="@+id/button1"
+            android:layout_width="match_parent"
+            android:layout_height="0dp"
+            android:layout_margin="5dp"
+            android:layout_weight="2"
+            android:background="@drawable/togglebuttona"
+            android:checked="false"
+            android:text='a'
+            android:textAllCaps="false"
+            android:textColor="#ffffff"
+            android:textOff='a'
+            android:textOn='a'
+            android:textSize="30dp"
+            android:visibility="gone" />
+
+        <ToggleButton
+            android:id="@+id/button2"
+            android:layout_width="match_parent"
+            android:layout_height="0dp"
+            android:layout_margin="5dp"
+            android:layout_weight="2"
+            android:background="@drawable/togglebuttonb"
+            android:checked="false"
+            android:text='b'
+            android:textAllCaps="false"
+            android:textColor="#ffffff"
+            android:textOff='b'
+            android:textOn='b'
+            android:textSize="30dp"
+            android:visibility="gone" />
+
+        <ToggleButton
+            android:id="@+id/button3"
+            android:layout_width="match_parent"
+            android:layout_height="0dp"
+            android:layout_margin="5dp"
+            android:layout_weight="2"
+            android:background="@drawable/togglebuttonc"
+            android:textAllCaps="false"
+            android:text='c'
+            android:textOn='c'
+            android:textOff='c'
+            android:textSize="30dp"
+            android:textColor="#ffffff"
+            android:visibility="gone" />
+
+        <ToggleButton
+            android:id="@+id/button4"

```

Comment:
You should use a more descriptive name for the id. This improves the readability of the code that refers to this id and makes it easier to identify which button is being used.